antidox/tester.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import json
+import urllib.parse
+import time
+import clean
+import datetime
+
+
+def wiki_write(result, header):
+    site = pywikibot.Site()
+    repo = site.data_repository()
+    page = pywikibot.Page(site, u"User_talk:" + args.wikibot)
+
+    heading = (header)
+    content = (result)
+    message = "\n\n{}\n{} --~~~~".format(heading, content)
+    page.save(summary="Testing", watch=None, minor=False, botflag=True,
+              force=False, async=False, callback=None,
+              apply_cosmetic_changes=None, appendtext=message)
+
+def log_change():
+  while True:
+    start = datetime.datetime.now() - datetime.timedelta(minutes=2)
+    rcstart = start.isoformat()
+    end = datetime.datetime.now()
+    rcend = end.isoformat()
+    page = ("http://sorenj02.nyc.corp.google.com/mediawiki/api.php?action=query&list=recentchanges&rclimit=500&rcprop=title%7Cids%7Csizes%7Cflags%7Cuser&rcdir=newer&rcstart="+ rcstart + "&rcend=now&format=json")
```

Comment:
If you use the UTC time, then using the timestamp for `rcend` in this URL instead of "now" should work.